api/HTMLMediaElement.json

```diff
@@ -3768,7 +3768,13 @@
               "version_added": "3.1"
             },
             "safari_ios": {
-              "version_added": "3"
+              "version_added": "3",
+              "partial_implementation": true,
+              "notes": [
+                "`volume` returns a value and is writeable; however, the value is always 1, and setting a value has no effect on the volume of the media object.",
+                "On iOS 12 and below, setting a value on `volume` has no effect on the value of `volume`.",
+                "On iOS 13 and up, setting a value on `volume` *will* update the value of `volume`, but it will be reset to 1 asynchronously."
+              ]
```

Comment:
I think we can skip these notes.  With the async reverts back to "1", it would be perceived as "no effect".

Reply:
I've taken them out.

My reasoning for the extra notes was that having the volume synchronously be changed only to be asynchronously reverted cost my quite a bit of time in debugging (it made me think the volume was working fine and that something else was resetting it).